components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -243,7 +250,8 @@ spec:
   ports:
   #should be 1 port
   - name: prometheus
-    port: 9090
+    port: 80
```

Comment:
now `80` is exposed so should also change it in

this should be changed to `80`
https://github.com/prometheus/prombench/blob/e4dbeb29b9ed4c1609516ef4d8e9f1d5c37537a8/components/prow/manifests/nginx-controller.yaml#L355-L357